modules/mysql/variables.tf

```diff
@@ -246,6 +246,7 @@ variable "additional_users" {
     name     = string
     password = string
     host     = string
+    type     = string
```

Comment:
Since adding a field to an object breaks backwards compatibility, could you convert this to a `list(map(any))`?

Reply:
Sure. I'll have it swapped first thing tomorrow morning